hooks/follower/usePutFollower.ts

```diff
@@ -0,0 +1,19 @@
+import { putFollower } from "@/api/putFollower";
+import { FollowerTypes } from "@/types/Follower";
+import { useMutation } from "@tanstack/react-query";
+import { AxiosError } from "axios";
+
+const usePutFollower = () => {
+  const mutation = useMutation({
+    mutationFn: ({ login, token }: FollowerTypes) => {
+      return putFollower(login, token);
+    },
+    onSuccess: () => {
+      location.reload();
+    },
+    onError: (err: AxiosError) => console.log(err),
+  });
+  return mutation;
+};
```

Comment:
[이 맥락과](https://github.com/SIMSIMS-JJANG/github-follow-detector/pull/5/files#r1436912987) 동일하게 전체를 return 해줘도 될 것 같습니당!
```suggestion
const usePutFollower = () => {
  return useMutation({
    mutationFn: ({ login, token }: FollowerTypes) => {
      return putFollower(login, token);
    },
    onSuccess: () => {
      location.reload();
    },
    onError: (err: AxiosError) => console.log(err),
  });
};
```